Add release notes for new features and enhancements (#8618)
wazuh/wazuh-kibana-app: docs/ref/release-notes.md

# Release notes

## Highlights

- New "[Security Analytics][security-analytics-module]" module [#7608](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7608). [[1][fork-security-analytics]]
  - Full lifecycle management for log normalization and event-based threat detection under a unified interface.

>   - Normalization module: manage decoders and KVDBs across four content spaces (Standard, Draft, Test, Custom) — full CRUD and duplication available in the Test space, read-only access in the Standard and Custom spaces [wazuh/wazuh-dashboard-security-analytics#22](https://github.com/wazuh/wazuh-dashboard-security-analytics/issues/22).

Reviewer: **question**: what do you mean by `duplication available in the Test space`?

Author: Resolved https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/c3a0c40372adb14d3257b39ba4ccae338f2a57fa

    - Decoders management with hierarchy relationship indicators (parent/child/sibling), space selector, and branch duplication options [wazuh/wazuh-dashboard-security-analytics#21](https://github.com/wazuh/wazuh-dashboard-security-analytics/issues/21).
    - KVDBs management with the same layout and space-aware interaction as Decoders [wazuh/wazuh-dashboard-security-analytics#23](https://github.com/wazuh/wazuh-dashboard-security-analytics/issues/23).
  - Integrations section as the main entry point: manage integration definitions backed by the Wazuh integrations index, with creation form and root decoder selection per space [wazuh/wazuh-dashboard-security-analytics#32](https://github.com/wazuh/wazuh-dashboard-security-analytics/issues/32).

>   - Detection section: write and manage Sigma-based detection rules that generate findings from normalized events.

Reviewer 2: The `Integrations` and `Spaces / Policies` share the same `Overview` section, but nothing is mentioned about the posibility to edit the `spaces` (only draft is editable but the changes can be promoted to test/custom), aside from the root decoder. 

**Suggestion:** an entry for `Overview` section could be added, with information about the spaces and `edit space` and edit the `Integrations` section.

Author: Resolved https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/4a54a6e8434768d5d4c69e5a34849c6dc0f8221f

  - Log Test tool to validate decoder and detection rule output against real or sample events.

>   - Filters management for pre-processing event pipelines.

Reviewer: ```suggestion
  - Filters management allows defining conditions to drop events in the event processing pipeline.
```

>   - Guided promotion workflow to move validated content from the Test space into the Custom (production) space.

Reviewer: ```suggestion
  - Move validated content between spaces using the promotion
```

- New "[Active Response][active-response-module]" module [#34606](https://github.com/wazuh/wazuh/issues/34606).
  - Dedicated section under **Explore > Active Responses** to create, edit, mute, and delete active responses — managed as a dedicated channel type, strictly separate from standard notification channels [wazuh/wazuh-indexer-notifications#3](https://github.com/wazuh/wazuh-indexer-notifications/issues/3) [wazuh/wazuh-dashboard-notifications#12](https://github.com/wazuh/wazuh-dashboard-notifications/issues/12).
  - Execution history browsable in Discover via the `wazuh-active-responses*` index pattern, provisioned automatically at startup [#8154](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8154).

>   - New "Add Active Response" action in Alerting monitor trigger configuration — only active response channels are shown, independently of standard notifications [wazuh/wazuh-dashboard-alerting#6](https://github.com/wazuh/wazuh-dashboard-alerting/issues/6).

Reviewer: **suggestion**: add reference to the **Active Response** monitor type.

```suggestion
  - New "Active Response" monitor type in Alerting that allows defining the new active response action using the "Add Active Response" button in Alerting monitor trigger configuration — only active response channels are shown, independently of standard notifications [wazuh/wazuh-dashboard-alerting#6](https://github.com/wazuh/wazuh-dashboard-alerting/issues/6).
```

>   - New "Add Active Response" action in Alerting monitor trigger configuration — only active response channels are shown, independently of standard notifications [wazuh/wazuh-dashboard-alerting#6](https://github.com/wazuh/wazuh-dashboard-alerting/issues/6).

Reviewer 2: **Suggestion:** An entry for the new Active Response dashboard could be added here. Suggestion below:

```suggestion
- New "[Active Response][active-response-module]" module [#34606](https://github.com/wazuh/wazuh/issues/34606).
  - Dedicated section under **Explore > Active Responses** to create, edit, mute, and delete active responses — managed as a dedicated channel type, strictly separate from standard notification channels [wazuh/wazuh-indexer-notifications#3](https://github.com/wazuh/wazuh-indexer-notifications/issues/3) [wazuh/wazuh-dashboard-notifications#12](https://github.com/wazuh/wazuh-dashboard-notifications/issues/12).
  - Execution history browsable in Discover via the `wazuh-active-responses*` index pattern, provisioned automatically at startup [#8154](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8154).
  - New "Add Active Response" action in Alerting monitor trigger configuration — only active response channels are shown, independently of standard notifications [wazuh/wazuh-dashboard-alerting#6](https://github.com/wazuh/wazuh-dashboard-alerting/issues/6).
  - New `Active Response Dashboard` under `Security operations > Active Response` to monitor `Active Responses` execution data.
```

Author: Resolved https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/6f4a59629b23201e7090f5de6188b65765093d2d

- Dashboard revamp for Wazuh 5.0 [#8031](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8031).
  - Dashboards and visualizations migrated to saved objects, enabling sharing, PDF generation, and embedding across all modules — regenerated automatically on startup if missing [#7815](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7815) [#7846](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7846) [#7919](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7919).

>   - Legacy PDF reporting system removed from the dashboard plugins; PDF and CSV generation now handled by the Wazuh Reporting fork [#7813](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7813).

Reviewer: ```suggestion
  - Legacy PDF reporting system removed from the dashboard plugins [#7813](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7813).
```

  - MITRE ATT&CK module redesigned to feed data from findings generated by Sigma rules [#8045](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8045).
  - File Integrity Monitoring module redesigned for the Wazuh 5.0 data model [#8044](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8044).
  - Malware Detection module redesigned for the Wazuh 5.0 data model [#8138](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8138).
  - Security Operations module redesigned for the Wazuh 5.0 data model (PCI DSS, GDPR, HIPAA, NIST 800-53, TSC) [#8139](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8139).
  - Dashboard modules migrated to consume findings index patterns [#8174](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8174).
  - Monitoring and Statistics dashboards adapted to the new metrics data streams [#8159](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8159).
  - Updated to UI theme v9 [wazuh/wazuh-dashboard#1052](https://github.com/wazuh/wazuh-dashboard/issues/1052).

>   - Sample data aligned with the Wazuh Common Schema v5 [#7839](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7839).

Reviewer: **thought**: I think it is outdated. The **Sample data** app is not available. I think we should add a new this was removed (app not registered, the code needs to be adapted).

Author: Resolved https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/c3a0c40372adb14d3257b39ba4ccae338f2a57fa

- Redesigned "[Health Check][healthcheck-module]" service [#7532](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7532).
  - Login no longer triggers a blocking health check — validation runs as a background service integrated with the server lifecycle [#7610](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7610).

>     - Initial startup blocks the service only when critical checks fail; index pattern validation and resource provisioning run at startup but do not block access.

Reviewer: **question**:  could you provide a reference where these resources provisioning is not blocking the initialization of the dashboard in the server side?

Author: Resolved https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/c3a0c40372adb14d3257b39ba4ccae338f2a57fa

    - Post-startup failures (e.g., API unavailable or a missing index pattern) surface as degraded status without blocking the rest of the dashboard.
    - Scheduled re-checks at a configurable interval (default 15 minutes, minimum 5 minutes) via `healthcheck.*` settings in `opensearch_dashboards.yml`.

>     - Validates and provisions required resources: index patterns, saved-object dashboards, notification channels, and API connectivity.

Reviewer: ```suggestion
    - Validates and provisions required resources: index patterns, saved-object dashboards. Validates: notification channels, manager API connectivity and compatibility and `run_as` configuration for the manager API hosts.
```

  - Dashboard components adapted to self-contain requirement failures — missing index patterns or unavailable API no longer blank the screen; affected views show contextual prompts instead [#7680](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7680).
  - New **Dashboard Management > Health Check** app displays overall status, per-check details, and JSON export for troubleshooting [#7697](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7697) [wazuh/wazuh-dashboard#870](https://github.com/wazuh/wazuh-dashboard/issues/870).
    - Platform header icon shows overall health state without interrupting navigation.
    - Refined check presentation: alerts-specific index pattern labeling, manager-aware API selector, warning-based messaging, and simplified filtering [#7744](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7744).
- Reporting revamp [wazuh/wazuh-indexer-reporting#45](https://github.com/wazuh/wazuh-indexer-reporting/issues/45). [[2][fork-reporting]]

>   - Fork of OpenSearch Reporting integrated into Wazuh Dashboard packages, with scheduled email delivery as the initial use case [wazuh/wazuh-dashboard#288](https://github.com/wazuh/wazuh-dashboard/issues/288) [wazuh/wazuh-dashboard#753](https://github.com/wazuh/wazuh-dashboard/issues/753).

Reviewer: ```suggestion
  - Fork of OpenSearch Reporting integrated into Wazuh dashboard packages, with scheduled email delivery as the initial use case [wazuh/wazuh-dashboard#288](https://github.com/wazuh/wazuh-dashboard/issues/288) [wazuh/wazuh-dashboard#753](https://github.com/wazuh/wazuh-dashboard/issues/753).
```

  - PDF and CSV reports from dashboards and saved searches, on-demand or on a schedule, with email delivery.
  - Wazuh-branded HTML email template for report notifications with download link [wazuh/wazuh-dashboard-reporting#46](https://github.com/wazuh/wazuh-dashboard-reporting/issues/46).

>   - Plugin adapted for Wazuh dashboard: white-labeled UI, Notebook-based reports removed, and report detail fixes [wazuh/wazuh-dashboard-reporting#54](https://github.com/wazuh/wazuh-dashboard-reporting/issues/54) [#51](https://github.com/wazuh/wazuh-dashboard-reporting/issues/51) [#50](https://github.com/wazuh/wazuh-dashboard-reporting/issues/50).

Reviewer: ```suggestion
  - Plugin adaptation: white-labeled UI, Notebook-based reports removed, and report detail fixes [wazuh/wazuh-dashboard-reporting#54](https://github.com/wazuh/wazuh-dashboard-reporting/issues/54) [#51](https://github.com/wazuh/wazuh-dashboard-reporting/issues/51) [#50](https://github.com/wazuh/wazuh-dashboard-reporting/issues/50).
```

- Fork of the Alerting and Notifications plugins. [[3][fork-alerting]] [[4][fork-notifications]]
  - Alerting and Notifications available under **Explore > Alerting** and **Explore > Notifications**.

>   - Health Check validates and provisions default notification channels (Slack, PagerDuty, Jira, Shuffle) and sample Alerting monitors on startup.

Reviewer: ```suggestion
  - Health Check validates the default notification channels (Slack, PagerDuty, Jira, Shuffle) and sample Alerting monitors on startup provided by indexer.
```

  - Multi-channel support: Slack, Microsoft Teams, Amazon Chime, Email (SMTP/SES), AWS SNS, and custom webhooks.
  - Active response channels kept as a dedicated channel type, separate from standard notification channels [#6](https://github.com/wazuh/wazuh-dashboard-alerting/issues/6).

## Breaking changes

- Wazuh Dashboard 4.x cannot be upgraded to 5.x. A fresh installation is required.

> - Upgrade to dashboard platform 3.x — plugins and configurations from 4.x (platform 2.x) are not compatible.

Reviewer: **thought**: I think we could remove this entry if this is defined:

```
Wazuh Dashboard 4.x cannot be upgraded to 5.x. A fresh installation is required.
```

Author: Resolved https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/c3a0c40372adb14d3257b39ba4ccae338f2a57fa

- `wazuh.yml` configuration file removed. All plugin settings have been migrated to `opensearch_dashboards.yml` and the Advanced Settings UI [#7871](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7871). See the [migration guide][migration-guide] for the full mapping.

>   - Several settings removed with no replacement: `customization.*`, `alerts.sample.prefix`, `ip.selector`, `ip.ignore`, `pattern`, `hideManagerAlerts`, and all monitoring and statistics job settings [#7871](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7871) [#7933](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7933) [#8102](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8102).

Reviewer: ```suggestion
  - Several settings removed with no replacement: `customization.*`, `alerts.sample.prefix`, `configuration.ui_api_editable`, `ip.selector`, `ip.ignore`, `pattern`, `hideManagerAlerts`, and all monitoring and statistics job settings [#7871](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7871) [#7933](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7933) [#8102](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8102).
```

- Default index pattern changed from `wazuh-alerts-*` to `wazuh-events-v5*`. The global index pattern selector has been removed from the navigation bar [#7933](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7933).

> - Dashboards redesigned as saved objects — inline dashboards from 4.x are not compatible with the new renderer. Custom dashboards must be exported before migration and re-imported afterwards [#7842](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7842).

Reviewer: ```suggestion
- Dashboards redesigned as saved objects — inline dashboards from 4.x are not compatible with the new renderer. Custom dashboards must be exported before migration and re-imported and adapted to the new data model afterwards [#7842](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7842).
```

- Health Check redesigned [#7532](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7532) — the blocking login-time health check (`/app/wazuh#/health-check`) and frontend routes `/health-check` and `/blank-screen` are removed [#7610](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7610). Individual `checks.*` toggles and `customization.logo.healthcheck` are replaced by `healthcheck.*` settings in `opensearch_dashboards.yml`. Health status is available at **Dashboard Management > Health Check** [#7697](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7697).
- Cluster mode mandatory for all installations [#7688](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7688) [wazuh/wazuh#31295](https://github.com/wazuh/wazuh/issues/31295) — the distinction between cluster and manager mode is removed; the dashboard always operates in cluster mode, including single-node deployments.

>   - All filters and search queries use `cluster.name` instead of `manager.name`.

Reviewer: ```suggestion
  - All filters and search queries use `wazuh.cluster.name` instead of `manager.name`.
```

>   - API calls route through cluster endpoints; `/cluster/status` checks and cluster-disabled UI components are removed.
>   - RBAC permissions updated: `manager:read`, `manager:update_config`, and `manager:restart` are replaced by `cluster:read`, `cluster:update_config`, and `cluster:restart`; `cluster:status` is no longer used.
> - Removed deprecated modules: OpenSCAP, CIS-CAT, and Osquery [#7645](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7645).

Reviewer 2: - `cluster:status` is still present on 5.x:

<img width="1913" height="690" alt="Image" src="https://github.com/user-attachments/assets/6e11a2d9-bd63-4453-b530-2ab4aee02c56" />

- In 4.x both `cluster:read`, `cluster:restart`, ... and `manager:read`, `manager:restart`, ... co-existed.
- `manager:read_api_config` and `cluster:read_api_config` existed too.

https://github.com/user-attachments/assets/725b5988-a786-45ca-b217-88879702675b

**Suggestion:** Re-phrase the comment to make clear that all those actions existed before, but the manager:* actions where deprecated and only the cluster:* actions remain.

```suggestion
  - API calls route through cluster endpoints; `/cluster/status` checks and cluster-disabled UI components are removed.
  - RBAC permissions updated: The legacy `manager:read`, `manager:update_config`, `manager:restart`, and `manager_read_api_config` have been consolidated into their existing `cluster:*` equivalents.
- Removed deprecated modules: OpenSCAP, CIS-CAT, and Osquery [#7645](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7645).
```

> - Legacy reporting application removed from the dashboard plugins. PDF and CSV generation is now handled by the Wazuh Reporting fork bundled with Wazuh Dashboard packages [#7813](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7813) [wazuh/wazuh-indexer-reporting#45](https://github.com/wazuh/wazuh-indexer-reporting/issues/45).

Reviewer: ```suggestion
- Legacy reporting application removed from the dashboard plugins. PDF and CSV generation is now handled by the Reporting fork bundled with Wazuh Dashboard packages [#7813](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7813) [wazuh/wazuh-indexer-reporting#45](https://github.com/wazuh/wazuh-indexer-reporting/issues/45).
```

- Rules, Decoders, CDB List, and Ruleset Test applications removed. Content management has moved to the [Wazuh Indexer Content Manager][content-manager-plugin] [#7901](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7901).

> - Backend monitoring and statistics jobs removed from the dashboard server [#7597](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7597). Agent and communications telemetry is now written to dedicated metrics data streams managed by the Wazuh Indexer.

Reviewer: **thought**: I am not sure who is the responsible to index the data, I guess this shoudl be the manager.

```suggestion
- Backend monitoring and statistics jobs removed from the dashboard server [#7597](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7597). Agent and communications telemetry is now written to dedicated metrics data streams managed by the Wazuh indexer.
```

- App Settings application removed — settings are now managed via `opensearch_dashboards.yml` or **Dashboard Management > Advanced Settings** [#7871](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7871).
- Sample Data, Docker, Statistics, and Cluster standalone applications removed [#8214](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8214) [#8215](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8215) [#8218](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8218) [#8220](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8220). Their functionality has been redistributed across the relevant modules and Server Management.

<!-- Links -->

[security-analytics-module]: ./modules/security-analytics/README.md
[active-response-module]: ./modules/active-response/index.md
[healthcheck-module]: ./modules/healthcheck.md
[migration-guide]: ./migration-4x-5x.md

> [content-manager-plugin]: https://wazuh.github.io/wazuh-indexer-plugins/ref/modules/content-manager/index.html

Reviewer: **issue**: this is referencing to GitHub pages and the reference is not versioned.

**suggestion**: you could reference to some issue or pull request where this feature was added.

Author: Resolved https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/c3a0c40372adb14d3257b39ba4ccae338f2a57fa

Reviewer: The reference was removed.

# Release notes

## Highlights

- New "[Security Analytics][security-analytics-module]" module [#7608](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7608). [[1][fork-security-analytics]]
  - Full lifecycle management for log normalization and event-based threat detection under a unified interface.
  - Normalization module: manage decoders and KVDBs across four content spaces (**Standard**, **Draft**, **Test**, **Custom**). **Draft**, **Test**, and **Custom** are user-managed spaces for authoring, validation, and production content; **Standard** holds built-in Wazuh content and is read-only [wazuh/wazuh-dashboard-security-analytics#22](https://github.com/wazuh/wazuh-dashboard-security-analytics/issues/22).
    - **Overview** section: unified entry point for integrations and **space policies** across all spaces. Create and edit integrations and configure space policies (enable status, root decoder) in **Draft**; **Test** and **Custom** display promoted content for validation and production [wazuh/wazuh-dashboard-security-analytics#32](https://github.com/wazuh/wazuh-dashboard-security-analytics/issues/32).

>     - Decoders management with hierarchy relationship indicators (parent/child/sibling), space selector, and **branch duplication** — copy a decoder and its dependent hierarchy as a new branch for isolated changes [wazuh/wazuh-dashboard-security-analytics#21](https://github.com/wazuh/wazuh-dashboard-security-analytics/issues/21).

Reviewer 3: `and **branch duplication** — copy a decoder and its dependent hierarchy as a new branch for isolated changes`

What does this refer to?

Author: I decided to clarify with another description here: https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/14e1f86f52d46dbf28042876e11da0bd1ed7cb5b

    - KVDBs management with the same layout and space-aware interaction as Decoders [wazuh/wazuh-dashboard-security-analytics#23](https://github.com/wazuh/wazuh-dashboard-security-analytics/issues/23).
  - Detection section: write and manage Sigma-based detection rules that generate findings from normalized events.
  - Log Test tool to validate decoder and detection rule output against real or sample events.
  - Filters management allows defining conditions to drop events in the event processing pipeline.
  - Guided promotion workflow to move validated content from **Draft** to **Test**, and from **Test** to **Custom**.
- New "[Active Response][active-response-module]" module [#34606](https://github.com/wazuh/wazuh/issues/34606).
  - Dedicated section under **Explore > Active Responses** to create, edit, mute, and delete active responses — managed as a dedicated channel type, strictly separate from standard notification channels [wazuh/wazuh-indexer-notifications#3](https://github.com/wazuh/wazuh-indexer-notifications/issues/3) [wazuh/wazuh-dashboard-notifications#12](https://github.com/wazuh/wazuh-dashboard-notifications/issues/12).

>   - Execution history browsable in Discover via the `wazuh-active-responses*` index pattern, provisioned automatically at startup [#8154](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8154).

Reviewer: **suggestion**: the `wazuh-active-response` index contains documents about the historical active response to execute. They do not provide information if the AR was executed or not. This could be misunderstood.

Author: I changed the description here: https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/9688c6632118d81a28a367fe197a17d1c3ae65d3

  - Case Management tab added to the Findings document details flyout [#8580](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8580) [#8589](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8589) [#8598](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8598).
  - **Cases** tab added to the Threat Hunting module [#8583](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8583) [#8608](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8608).
  - Dashboard modules migrated to consume findings index patterns [#8174](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8174).

>   - Monitoring and Statistics dashboards adapted to the new metrics data streams [#8159](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8159).

Reviewer: **nitpick**: Monitoring dashboard did not exist previously.

```suggestion
  - Statistics dashboard adapted to the new metrics data streams [#8159](https://github.com/wazuh/wazuh-dashboard-plugins/issues/8159).
```

  - Health Check validates the default notification channels (Slack, PagerDuty, Jira, Shuffle) and sample Alerting monitors on startup provided by indexer.
  - Multi-channel support: Slack, Microsoft Teams, Amazon Chime, Email (SMTP/SES), AWS SNS, and custom webhooks.
  - Active response channels kept as a dedicated channel type, separate from standard notification channels [#6](https://github.com/wazuh/wazuh-dashboard-alerting/issues/6).

## Additional highlights

- SSL certificate support for Wazuh manager API connections — client certificate authentication and CA verification via `key`, `cert`, and `ca` host settings; **Verify CA** column in the API Connections table [#8015](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8015) [#8212](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8212). See [Configuration](./configuration.md#define-wazuh-server-hosts).
- Indexer management **Settings** section in Server Management [#8206](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8206).
- CTI content update button to request on-demand threat intelligence content updates [#8201](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8201).

> - CTI Console registration flow — UI and registration status API [#8486](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8486) [#7663](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7663).

Reviewer: **nitpick**: move to **Hightlights** section

```suggestion
- Added ability to configure SSL for manager API host communications [#8015](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8015) [#8212](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8212). See [Configuration](./configuration.md#define-wazuh-server-hosts).
- Added **Settings** app to Indexer management section [#8206](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8206).
- Added CTI content update button to request on-demand threat intelligence content updates [#8201](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8201).
- Added CTI Console registration flow — UI and registration status API [#8486](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8486) [#7663](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7663).
```

- Backend monitoring and statistics jobs removed from the dashboard server [#7597](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7597). Agent and communications telemetry is now written to dedicated metrics data streams managed by the Wazuh indexer.
- App Settings application removed — settings are now managed via `opensearch_dashboards.yml` or **Dashboard Management > Advanced Settings** [#7871](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7871).
- Default value of `wazuh.updates.disabled` changed from `false` to `true` — update notifications are disabled by default in Advanced Settings [#8236](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8236).
- Only one Wazuh server API host configuration is allowed per indexer [#8436](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8436).
- Sample Data, and Cluster standalone applications removed [#8214](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8214) [#8220](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8220). The views of cluster has been redistributed to the Status app.

<!-- Links -->

> [security-analytics-module]: ./modules/security-analytics/README.md

Reviewer 4: This link doesn't work:
`./modules/security-analytics/README.md`
please change it to: 
`./modules/security-analytics/index.md`

Author: resolved here:; https://github.com/wazuh/wazuh-dashboard-plugins/pull/8618/changes/698e1755bc2d057f47aaaab5ced350e0cc26b595

> - Sample Data, Docker, Statistics, and Cluster standalone applications removed [#8214](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8214) [#8215](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8215) [#8218](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8218) [#8220](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8220). The Sample Data app is no longer registered in Wazuh 5.x; internal sample data generators were adapted to the Wazuh Common Schema v5 [#7839](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7839), but end users can no longer load sample data from the UI. Remaining functionality has been redistributed across the relevant modules and Server Management.

Reviewer: ```suggestion
- Sample Data, and Cluster standalone applications removed [#8214](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8214) [#8220](https://github.com/wazuh/wazuh-dashboard-plugins/pull/8220). The views of cluster has been redistributed to the Status app.
```

Docker and Statistics apps are present. They were removed temporally but they were added later.
